src/pages/config/InsightsDetailsChangesPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { useParams, useSearchParams } from "react-router-dom";
```

Comment:
Let's rename this to something like ConfigDetailsInsights instead; adding changes to the name is a bit confusing